rest-api-spec/src/main/resources/rest-api-spec/api/cat.nodes.json

```diff
@@ -43,7 +43,11 @@
       },
       "local":{
         "type":"boolean",
-        "description":"Return local information, do not retrieve the state from master node (default: false)"
+        "description":"Return local information, do not retrieve the state from master node (default: false)",
```

Comment:
```suggestion
        "description":"Calculate the selected nodes using the local cluster state rather than the state from master node (default: false)",
```